src/index.js

```diff
@@ -18,18 +20,25 @@ const game = {
   },
   gameFailSequence() {
     this.canvas.removeEventListener(EVENTS.LEVEL_FAIL, this._gameFailSequence);
-    console.log('Yo, don\'t suck');
     audio.die()
     this.engine.stop();
     // TODO - Display Retry Screen
     setTimeout(() => window.location.reload(), 3000);
+  },
+  frame() {
```

Comment:
What's the purpose of this function?

Reply:
debugging... literally needed to walk frames to measure how to calc the gravity :)